optuna/study.py

```diff
@@ -482,6 +484,23 @@ def _create_record_and_aggregate_column(trial):
 
         return df
 
+    @experimental('1.2.0')
+    def enqueue_trial(self, params):
+        # type: (Dict[str, Any]) -> None
+        """Enqueue a trial with given parameter values.
+
+        You can fix the next sampling parameters which will be evaluated in your
+        objective function.
+
+        Args:
+            params:
+                Parameter values to pass your objective function.
```

Comment:
I believe we could elaborate a bit on the structure of `params`, or include an example. We can address that later though.